src/main.rs

```diff
@@ -222,7 +222,7 @@ pub struct TopLevel {
     vdpa: Vec<String>,
 
     #[argh(option, long = "vsock")]
```

Comment:
You shouldn't add my `Signed-off-by:` it's not passed through any of my trees.

Reply:
Okay :)
I'm not sure when to add reviewers' `Signed-off-by`, feel free to add your `Signed-off-by` when you need 😄